packages/protocol/contracts/ERC1724/base/ZkAssetMintableBase.sol

```diff
@@ -115,7 +120,7 @@ contract ZkAssetMintableBase is ZkAssetOwnableBase {
         bytes memory outputNotes,
         address publicOwner,
```

Comment:
Should `confidentialTransferFrom()` for the mintable ZkAsset also not have some form of permissioning around the type of proof it accepts? Should it check that the passed `uint24 _proof` is supported by the asset, like:
```
        bool result = supportsProof(_proofId);
        require(result == true, "expected proof to be supported")
```

We are overwriting and redefining `confidentialTransferFrom()` here, so the checks in the underlying `confidentialTransferFrom()` on the `ZkAssetOwnableBase.sol` are not being performed. 

We don't need to do this for the `ZkAssetBurnableBase.sol`  - it doesn't redefine the transfer methods

Reply:
Yep you're right!